Assets/Scripts/GroupSelectionWizard.cs

```diff
@@ -0,0 +1,275 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+
+public class GroupSelectionWizard : MonoBehaviour
+{
+    public GameObject ButtonPanel;
+    public GameObject smallOrangeButtonPrefab;
+    public GameObject NextButton;
+    public TMPro.TextMeshProUGUI DirectionsText;
+    public GameObject NextScreen;
+   
+    private List<Action> steps;
+    private int currentStep;
+    private List<GameObject> buttons;
+ 
+    // The user record:
+    private UserRecord userRecord;
+
+    // Start is called before the first frame update
+    private void Start()
+    {
+        userRecord = new UserRecord();
+        currentStep = 0;
+        steps = new List<Action>();
+        steps.Add(ShowGroups);
+        steps.Add(ShowColors);
+        steps.Add(ShowAnimals);
+        steps.Add(ShowNumbers);
+        //steps.Add(ShowNetworks);
+        buttons = new List<GameObject>();
+        ShowGroups();
+        Button nextButton = NextButton.GetComponent<Button>();
+        nextButton.onClick.AddListener(NextStep);
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+
+
+    public void ShowGroups()
```

Comment:
As we discussed in the office, there is some repetition here, but honestly I think that's fine for now. It's readable and it works!